extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -1532,6 +1580,98 @@ describes.realWin('amp-a4a', {amp: true}, (env) => {
       );
     });
 
+    it('should fallback to non-AMP creative if shadow dom not supported', async () => {
+      const fixture = await createIframePromise();
```

Comment:
I think I moved these all inside `describes` as part of fixit, do we still need these fixtures, or can we use env.win + friends?

Reply:
We should switch them to `env.win`. But for now I just added this test to be consistent with other tests to reuse the setup code.